.github/workflows/ci.yml

```diff
@@ -118,6 +118,127 @@ jobs:
             test/functional.test.exe
             test/runhcs.test.exe
             test/sample-logging-driver.exe
+  
+  integration-tests:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [windows-2019, windows-2022]
+    
+    steps:
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '1.18.3'
+          check-latest: true
+      - name: Set env
+        shell: bash
+        run: |
+          mkdir -p "${{ github.workspace }}/bin"
+
+          echo "GOPATH=${{ github.workspace }}" >> $GITHUB_ENV
+          echo "${{ github.workspace }}/bin" >> $GITHUB_PATH
+          echo "${{ github.workspace }}/src/github.com/containerd/containerd/bin" >> $GITHUB_PATH
+      - uses: actions/checkout@v2
+        with:
+          path: src/github.com/containerd/containerd
+          repository: 'containerd/containerd'
+
+      - uses: actions/checkout@v2
+        with:
+          path: src/github.com/Microsoft/hcsshim
+
+      - name: Build binaries
+        shell: bash
+        working-directory: src/github.com/containerd/containerd
+        run: |
+          set -o xtrace
+          mingw32-make.exe binaries
+          script/setup/install-cni-windows
+
+      - name: Build the shim
+        working-directory: src/github.com/Microsoft/hcsshim
+        shell: powershell
+        run: |
+          go build -mod vendor -o "${{ github.workspace }}/src/github.com/containerd/containerd/bin/containerd-shim-runhcs-v1.exe" .\cmd\containerd-shim-runhcs-v1
+
+      - name: Get crictl tool
+        shell: powershell
+        run: |
+          curl.exe -L "https://github.com/kubernetes-sigs/cri-tools/releases/download/v1.24.2/crictl-v1.24.2-windows-amd64.tar.gz" -o c:\crictl.tar.gz
+          tar xf c:\crictl.tar.gz -C "${{ github.workspace }}/bin"
+
+      - name: Run containerd integration tests
+        shell: bash
+        working-directory: src/github.com/containerd/containerd
+        run: |
+          export EXTRA_TESTFLAGS="-timeout=20m"
+          make integration
+
+      - name: Run containerd CRI integration tests
+        shell: bash
+        working-directory: src/github.com/containerd/containerd
+        env:
+          TEST_IMAGE_LIST: ${{github.workspace}}/repolist.toml
+          BUSYBOX_TESTING_IMAGE_REF: "k8s.gcr.io/e2e-test-images/busybox:1.29-2"
+          RESOURCE_CONSUMER_TESTING_IMAGE_REF: "k8s.gcr.io/e2e-test-images/resource-consumer:1.10"
+          CGO_ENABLED: 1
+        run: |
+          cat > "${{ env.TEST_IMAGE_LIST }}" << EOF
+          busybox = "${{ env.BUSYBOX_TESTING_IMAGE_REF }}"
+          ResourceConsumer = "${{ env.RESOURCE_CONSUMER_TESTING_IMAGE_REF }}"
+          EOF
+
+          FOCUS="[^TestContainerdRestart$]" make cri-integration
+
+#      - name: Install containerd service
```

Comment:
Probably leave a comment stating that we'll re-enable these when the hostprocess and other kinks are ironed out, but this is just a reminder